### PR TITLE
Datepicker input box support

### PR DESCRIPTION
**Pull request by Blackbaud-PaulCrowder**

### CODE REVIEW

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-datetime?_sp.spa=pr-input-box-support](https://developer.blackbaud.com/skyux-datetime?_sp.spa=pr-input-box-support)

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-datetime?_sp.spa=pr-input-box-support](https://developer.blackbaud.com/skyux-datetime?_sp.spa=pr-input-box-support)

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-datetime?_sp.spa=pr-input-box-support](https://developer.blackbaud.com/skyux-datetime?_sp.spa=pr-input-box-support)

File: src/app/public/modules/datepicker/datepicker.component.scss [hunk 1]
```diff
@@ -1,4 +1,5 @@
 @import "~@skyux/theme/scss/mixins";
+@import "~@skyux/theme/scss/variables";
```
Comment (Blackbaud-AlexKingman) [inline src/app/public/modules/datepicker/datepicker.component.scss:2]: Was this imported by accident? I don't see any changes to the scss file other than this line.

File: src/app/public/modules/datepicker/datepicker.component.spec.ts [hunk 2]
```diff
@@ -208,14 +242,16 @@ describe('datepicker', () => {
     let component: DatepickerTestComponent;
     let nativeElement: HTMLElement;
 
-    beforeEach(() => {
+    beforeEach(fakeAsync(() => {
```
Comment (Blackbaud-AlexKingman) [inline src/app/public/modules/datepicker/datepicker.component.spec.ts:245]: To help explain this to future devs, would you mind a comment on why the `fakeAsync` is necessary here?

File: src/app/public/modules/datepicker/datepicker.component.html [hunk 3]
```diff
@@ -1,34 +1,45 @@
-<div
-  class="sky-datepicker"
->
+<div class="sky-datepicker">
```
Comment (Blackbaud-AlexKingman) [inline src/app/public/modules/datepicker/datepicker.component.html:1]: All html attributes should be on a new line.

**Comment by Blackbaud-AlexKingman** [comment]
One last question... In modern theme, the calendar is much larger than before. I know our styles intentionally add a bit more white space, but just wanted to point out this scenario makes it more apparent.

Before:
Screen Shot 2020-06-15 at 2 17 14 PM

After:
Screen Shot 2020-06-15 at 2 17 22 PM

Is this an issue? @Blackbaud-ToddRoberts

**Comment by Blackbaud-ToddRoberts** [comment]
@Blackbaud-AlexKingman yes problematic, the intent was for the calendar picker to be the same size/font size/spacing as in default theme

**Comment by Blackbaud-PaulCrowder** [comment]
> Couple minor style things and one question: Using the preview build link above, it looks like the datepicker button still has left rounded corners when using modern theme. Did the styles not get updated?
> 
> Screen Shot 2020-06-15 at 2 12 05 PM

Currently there's no use case for a datepicker in modern theme outside an input box. Once there is, I can circle back and style it.

**Comment by Blackbaud-PaulCrowder** [comment]
> One last question... In modern theme, the calendar is much larger than before. I know our styles intentionally add a bit more white space, but just wanted to point out this scenario makes it more apparent.
> 
> Before:
> Screen Shot 2020-06-15 at 2 17 14 PM
> 
> After:
> Screen Shot 2020-06-15 at 2 17 22 PM
> 
> Is this an issue? @Blackbaud-ToddRoberts

This PR is just for the datepicker input and button display properly inside an input box. There's more work to be done around the calendar component and dropdowns in general; once I get to that part, I can address the button spacing.

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-datetime?_sp.spa=pr-input-box-support](https://developer.blackbaud.com/skyux-datetime?_sp.spa=pr-input-box-support)

**Pull request merged**